src/test/suite/format/samples/indent-code-after-sharp-directive-false.in.ahk

```diff
@@ -0,0 +1,4 @@
+#IfWinActive WinTitle
```

Comment:
Let's add another small sample with this option enabled. I know we already have existing code for this, but smaller samples are much easier to debug :)

Reply:
Done.